migration/docker/Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM --platform=linux/amd64 eclipse-temurin:21
+ARG KAFKA_VERSION="3.4.0"
+ARG KAFKA_CONNECT_VERSION="${KAFKA_VERSION}"
+ARG KAFKA_SCALA_VERSION="2.13"
+ARG PUBSUB_GROUP_KAFKA_CONNECTOR_VERSION="1.2.0"
+ARG KAFKA_HOME_ROOT="/opt"
+ENV KAFKA_HOME="${KAFKA_HOME_ROOT}/kafka"
+ARG KAFKA_CONFIG_DIR="${KAFKA_HOME}/config"
+ARG KAFKA_RELEASE="kafka_${KAFKA_SCALA_VERSION}-${KAFKA_VERSION}"
+ARG KAFKA_TARBALL="${KAFKA_RELEASE}.tgz"
+ARG KAFKA_DOWNLOAD_URL="https://archive.apache.org/dist/kafka/${KAFKA_VERSION}/${KAFKA_TARBALL}"
+# The pubsub-group-kafka-connector file needs to be pre-built/downloaded using maven or other similar tool.
+# References:
+# 1) https://github.com/googleapis/java-pubsub-group-kafka-connector/releases/
+# 2) https://central.sonatype.com/artifact/com.google.cloud/pubsub-group-kafka-connector
+ARG PUBSUB_GROUP_KAFKA_CONNECTOR_JAR="pubsub-group-kafka-connector-${PUBSUB_GROUP_KAFKA_CONNECTOR_VERSION}.jar"
+ARG KAFKA_CONNECT_CONFIGURE_SCRIPT="configure-kafka-connect.sh"
+ARG BUILD_KAFKA_CONNECT_STARTUP_SCRIPT="start-kafka-connect.sh"
```

Comment:
why BUILD_ prefix?

Reply:
There is the location relative to the script, and the location within the docker image. BUILD is the first